libpod/runtime_img.go

```diff
@@ -613,7 +613,17 @@ func (r *Runtime) PushImage(source string, destination string, options CopyOptio
 	// Get the destination Image Reference
 	dest, err := alltransports.ParseImageName(destination)
 	if err != nil {
-		return errors.Wrapf(err, "error getting destination imageReference for %q", destination)
+		// add the docker:// transport to see if they neglected it.
+		if strings.Contains(destination, "://") {
```

Comment:
Depending on how you want to handle this @TomSweeneyRedHat , there is a slice for transports and we have a util.StringInSlice which takes a string and slice and returns a book depending on whether the string can be found in slice.

Reply:
or maybe it is time we break down and make an image-related function that is something like hasTransport ?

Reply:
I've made hasTransport, the stringinslice didn't fit as I've only a simple string at this point.